common-lib/src/cmr/common/util.clj

```diff
@@ -1032,3 +1032,9 @@
     (if (or (empty? x) (last x))
       x
       (recur (drop-last x)))))
+
+(defn tee
```

Comment:
This is not necessary and should not be used like this. If there is a debug statement needed the existing logging functionality should be used